sdk/resourcemanager/operationalinsights/armoperationalinsights/zz_generated_models.go

```diff
@@ -1130,8 +1581,8 @@ type Workspace struct {
 	// REQUIRED; The geo-location where the resource lives
 	Location *string `json:"location,omitempty"`
 
-	// The etag of the workspace.
-	Etag *string `json:"etag,omitempty"`
+	// The ETag of the workspace.
+	ETag *string `json:"eTag,omitempty"`
```

Comment:
This is unfortunate.  We should have normalized this to `ETag` in the code generator but it's too late now as that would likely introduce breaking changes elsewhere.